Example/Emission/AppSetup.m

```diff
@@ -53,7 +53,7 @@ - (instancetype)initWithAmbientContext
       _emissionLoadedFromString = [NSString stringWithFormat:@"PR #%@", @(prNumber)];
 
     } else if (useRNP) {
-      NSString *rnpString = [NSString stringWithFormat:@"http://%@:8081/Example/Emission?platform=ios&dev=true", packagerURL];
+      NSString *rnpString = [NSString stringWithFormat:@"http://%@:8081/Example/Emission/index.ios.bundle?platform=ios&dev=true", packagerURL];
```

Comment:
@orta This wasn’t working.

Reply:
ah yeah, I fixed it in my PR, but expected it to have got through to master - thought this PR gonna be inactive long enough